### PR TITLE
Log memory usage and use fmt.Printf to do info level log

### PR DESCRIPTION
**Pull request by shifucun**
Golang GCP logging level: https://cloud.google.com/functions/docs/monitoring/logging.

In longer term, we could consider using logging client. For now, separate info and error level would be good.

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -99,7 +105,7 @@ func main() {
 	if err != nil {
 		log.Fatalf("failed to listen on network: %v", err)
 	}
-	log.Println("mixer ready to server")
+	fmt.Println("mixer ready to server")
```
Comment (beets) [inline main.go:108]: nit: "mixer ready to serve" (extra r at the end)

**Review by beets — APPROVED** [review]

**Comment by beets** [comment]
it's very subtle that log / fmt ends up logged differently. should we standardize on fmt, and specify the severity?

File: main.go (hunk 1 above)
Comment (shifucun) [reply main.go]: Done

File: util/util.go [hunk 2]
```diff
@@ -268,3 +269,19 @@ func RemoveKeyPrefix(key string) (string, error) {
 	}
 	return match[3], nil
 }
+
+// PrintMemUsage outputs the current, total and OS memory being used. As well as the number
+// of garage collection cycles completed.
```
Comment (pradh) [inline util/util.go:274]: We could also consider using https://cloud.google.com/profiler/docs/profiling-go

**Review by pradh — APPROVED** [review]

**Comment by shifucun** [comment]
> it's very subtle that log / fmt ends up logged differently. should we standardize on fmt, and specify the severity?

Looked a bit more on https://golang.org/pkg/fmt/, so the fmt.Errorf does not log to stderr. I think long term we should use the cloud logging client. Need to give it a try and have some feeling on how much overhead it adds to the code.

File: util/util.go (hunk 2 above)
Comment (shifucun) [reply util/util.go:274]: Good point, added to https://github.com/datacommonsorg/mixer/issues/347 and can do for the next PR

**Pull request merged**